terraform/prd/tokyo/main.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    region = "ap-northeast-1"
+    bucket = "terraform-remote-state-smartnews-prd-ap-northeast-1"
+    key    = "realtime-streaming/realtime-streaming-llm-ops.tfstate"
```

Comment:
`realtime-streaming` is the system value from ops-bootstrap. If you are using a different system, please use that value.

Reply:
This service deploy under common-0 tokyo prd realtime-stremaing, so I guess should use this system?